package.json

```diff
@@ -45,10 +47,21 @@
     "jest": "^28.0.0",
     "jsdom": "^19.0.0",
     "markdown-it": "^13.0.0",
+    "prettier": "2.7.1",
     "punycode": "^2.1.1",
     "ts-jest": "^28.0.0",
     "typescript": "^4.4.2"
   },
+  "prettier": {
+    "parser": "typescript",
+    "singleQuote": true,
+    "jsxSingleQuote": true,
+    "semi": false,
+    "tabWidth": 2,
+    "trailingComma": "none",
+    "bracketSpacing": true,
+    "arrowParens": "always"
+  },
```

Comment:
Do we want to move that into it's own file? That's how it's done in the other HedgeDoc repos…